rfc/RFC0001.md

```
@@ -0,0 +1,364 @@
+
+**Example Peer Reject Payload**
+
+```json
+{
+    "action": "PeerReject",
+    "sourceNode": "https://barcodes.example.com",
+    "message": "Peering rejected. This node does not add additional peers."
+}
+```
+
+**Example Peer Pending Payload**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)